dom/manipulate.ts

```diff
@@ -29,7 +29,7 @@ export type InsertableElement = string | Element | Element[];
  * @param {string} html
  * @returns {HTMLElement}
  */
-function parseHtml (html : string) : HTMLElement
+function parseHtml<T extends HTMLElement = HTMLElement> (html : string) : T
```

Comment:
This is an internal function, and we don't get anything from making it generic. We can just cast afterwards.